[2025-02-26T14:19:51.518+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-26T14:19:51.546+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2021-09-02T00:00:00+00:00 [queued]>
[2025-02-26T14:19:51.564+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2021-09-02T00:00:00+00:00 [queued]>
[2025-02-26T14:19:51.565+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-26T14:19:51.760+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): first_task> on 2021-09-02 00:00:00+00:00
[2025-02-26T14:19:51.781+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag', 'first_task', 'scheduled__2021-09-02T00:00:00+00:00', '--job-id', '112', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp65wcvtiz']
[2025-02-26T14:19:51.782+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=3568) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-26T14:19:51.783+0000] {standard_task_runner.py:105} INFO - Job 112: Subtask first_task
[2025-02-26T14:19:51.783+0000] {standard_task_runner.py:72} INFO - Started process 3590 to run task
[2025-02-26T14:19:51.862+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag.first_task scheduled__2021-09-02T00:00:00+00:00 [running]> on host 25887b48f6fb
[2025-02-26T14:19:51.992+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2021-09-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-09-02T00:00:00+00:00'
[2025-02-26T14:19:51.994+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-26T14:19:51.994+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-26T14:19:51.995+0000] {logging_mixin.py:190} INFO - Current task name:first_task state:running start_date:2025-02-26 14:19:51.547247+00:00
[2025-02-26T14:19:51.995+0000] {logging_mixin.py:190} INFO - Dag name:our_first_dag and current dag run status:running
[2025-02-26T14:19:51.995+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-26T14:19:51.996+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-26T14:19:51.997+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2025-02-26T14:19:52.017+0000] {subprocess.py:99} INFO - Output:
[2025-02-26T14:19:52.020+0000] {subprocess.py:106} INFO - hello world, this is the first task!
[2025-02-26T14:19:52.021+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-26T14:19:52.054+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-26T14:19:52.055+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=first_task, run_id=scheduled__2021-09-02T00:00:00+00:00, execution_date=20210902T000000, start_date=20250226T141951, end_date=20250226T141952
[2025-02-26T14:19:52.173+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-26T14:19:52.173+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-26T14:19:52.173+0000] {logging_mixin.py:190} INFO - Dag name:our_first_dag queued_at:2025-02-26 14:19:50.056001+00:00
[2025-02-26T14:19:52.174+0000] {logging_mixin.py:190} INFO - Task hostname:25887b48f6fb operator:BashOperator
[2025-02-26T14:19:52.200+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-26T14:19:52.249+0000] {taskinstance.py:3901} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-02-26T14:19:52.253+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
